source/clear-linux/guides/network/network-bonding.rst

```diff
@@ -37,19 +47,15 @@ below shows how to configure systemd to use the ``bonding`` driver.
       MIIMonitorSec=1s
       LACPTransmitRate=fast
 
-   The syntax for this file is defined in the systemd.netdev_ manpage.
-   `This example`__ may be used verbatim, or tuned to your particular
-   requirements.  Note that ``802.3ad`` mode requires explicit support from
-   your NICs and network switch. This and other modes may also require
-   additional configuration of your network switch.
+   Refer to the systemd.netdev_ manpage for :file:`30-bond1.netdev` file
+   syntax. This example is based on Example 9 on the manpage. Modify the
+   example for your configuration.
 
-__ https://www.freedesktop.org/software/systemd/man/systemd.netdev.html#id-1.20.10
```

Comment:
My intent here was to link to the specific anchor in the manpage, but it looks like that anchor has changed to https://www.freedesktop.org/software/systemd/man/systemd.netdev.html#id-1.24.10

Maybe that's just not reliable.

Reply:
You're correct, it's better to link to the manpage itself in case the anchor changes again.